src/forms/SelectField.js

```diff
@@ -6,36 +6,31 @@ import Select from '../ui/Select';
 import FormField from './FormField';
 
 /**
- * Component that encapsulates the `react-select` component
+ * Component that encapsulates the `react-select` component.
  */
 const SelectField = ({
   input,
   label,
   defaultValue,
   help,
   meta,
-  options,
   valueKey,
-  isLoading,
-  clearable,
   multi,
-  creatable,
-  async,
   onChangeValue,
   innerRef,
   onChangeWithValue = true,
   denormalize,
   ...props
 }) => {
-  const denormalized = (denormalize ? denormalize(input.value) : input.value) || defaultValue;
+  const denormalized = (!_.isNil(denormalize) ? denormalize(input.value) : input.value) || defaultValue;
```

Comment:
could you put a comment here to explain what this code is doing?

Reply:
@twheys sure. That came from dashmore's select field btw. That said there is no usage of it on Bandit. So, if you're not using it in Dashmore I'd rather remove the prop.